modules/rostests/tests/DxHalTest/main.c

```diff
@@ -99,15 +99,15 @@ int WINAPI WinMain (HINSTANCE hInst, HINSTANCE hPrevInst,
     DdResetVisrgn            = (BOOL (APIENTRY*)(LPDDRAWI_DDRAWSURFACE_LCL, HWND))GetProcAddress(lib, "GdiEntry6");
 
 	printf("This apps showing how to start up directx draw/d3d interface and some other as well\n");
-	printf("This code have been releae to some close applactons with my premtions, if any company\n");
-    printf("want use part or whole code, you need contact the orginal author to ask for premtions\n");
-    printf("This code are release under alot of diffent licen\n");
-    printf("All GPL and LGPL project have right use and studing this code.\n");
-    printf("This code maybe need more comment to known how stuff working and maybe looking bit mesy\n");
-	printf("Bestreagds Magnus Olsen magnus@greatlord.com or greatlord@reactos.org\n");
+	printf("This code have been released to some close applications with my promotions, if any company\n");
```

Comment:
premtions seems to mean permission. (same as in line 103)

Reply:
s/have/has

Reply:
>s/have/has
Where is it?

Reply:
"my promotions" changed to "my permission"

Reply:
> > s/have/has
> > Where is it?

This code have been - - > this Code has been